mu-trees/tests/unit/merge-addons-config-test.js

```diff
@@ -0,0 +1,74 @@
+import { module, test } from "qunit";
+import mergeAddonsConfig from "ember-resolver/merge-addons-config";
+
+module("ember-resolver/merge-addons-config");
+
+const emptyConfig = function() {
+  return { types: {}, collections: {} };
+};
+
+test("Trigger error if two addons configure the same type", function(assert) {
```

Comment:
this may either encourage addon authors to use _very_ unique types, or maybe it'll encourage them to use existing types. :) (I'm hoping for reuse)

Reply:
Probably, it is also something that needs confirmation.